examples/basic-webpack/index.jsx

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ExampleApplication = React.createClass({
+  render: function() {
+    var elapsed = Math.round(this.props.elapsed  / 100);
+    var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
+    var message = 'React has been successfully running for ' + seconds + ' seconds.';
+
+    return (<p>{message}</p>);
+  }
+});
+
+var start = new Date().getTime();
+
+setInterval(function() {
```

Comment:
Also, this is meant to be the same example as the others, which are admittedly not the best examples you could ever write.

`setState` is also not an antipattern. There's a big enough perception in the community that it is and we need to not add support to that sentiment.